src/main/java/org/opensearch/ad/MemoryTracker.java

```diff
@@ -238,42 +235,66 @@ public long estimateTotalModelSize(AnomalyDetector detector, int numberOfTrees,
      *      - SmallNodeStore (small node store since our sample size is 256, less than the max of short): 6120
      *      + BoxCacheFloat
      *        - other: 104
-     *        - BoundingBoxFloat: (1040 + 255* (dimension * 4 * 2 + 64)) * adjusted bounding box cache usage,
-     *           where if full we have 255 inner node and each box has 80 bytes.
-     *           Plus metadata, we can have in total 21544 bytes.
-     *           {@code adjusted bounding box cache usage = (bounding box cache fraction >= 0.3? 1: bounding box cache fraction)}
-     *           {@code >= 0.3} we will still initialize bounding box cache array of the max size,
-     *           but exclude them using the cache ratio.  It is not guaranteed we will only
-     *           use cache ratio in the array.  For example, with cache ratio 0.5, we used 150
-     *           out of 255 elements.  So can have two float array whose size is the number of
-     *           dimensions; other constants are the metadata size.
+     *        - BoundingBoxFloat: (1040 + 255* (dimension * 4 + 16) * 2 + 32)) * actual bounding box cache usage,
+     *           {@code actual bounding box cache usage = (bounding box cache fraction >= 0.3? 1: bounding box cache fraction)}
```

Comment:
Why is there a ? after 0.3?

Reply:
It is the ternary operator in java:

booleanExpression ? expression1 : expression2